boa/boa-id-v1.jsonld

```diff
@@ -0,0 +1,35 @@
+{
+  "@context":
+  {
+    "@version": 1.1,
+    "bid": "https://sphereon-opensource.github.io/factom-vc-contexts/boa/boa-id-v1#",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+    "schema": "http://schema.org/",
+
+    "BoaIdCredential" : "bid:BoaIdCredential",
+
+    "issueDate": {
```

Comment:
Assumption is that the ID is valid from the issueDate? Because I could issue an ID that becomes valid in the future for instance. Probably quite a common use case for onboarding new BOAs. Since this is such a simple value/distinction I would include it now tbh. IssueDate can be 'now' for now

Reply:
There is an `issuanceDate` on the credential as well, so the VC can be issued for a BOA before it becomes valid (which I assume is necessary in some circumstances). This value would be in the `credentialSubject` and represent the date from which the id is valid. Is this what you mean?

Reply:
Not entirely. I get that you can get it from the credential itself as well. But since we have issueDate in here it suggest that this is the date it is issued (so basically the issuanceDate from the credential). Se remark above. I rather like a start and end date for validity as they are different data then the issuanceDate

Reply:
You want to have simple logic. _Is this BOA ID valid on this date?_ _When will this BOA ID become invalid?_

Reply:
Okay, I've updated the `validityDuration` to an `expirationDate`. We can discuss what is needed there, but I'm assuming that this won't be a problem.

Reply:
Would we also want to make the `issueDate` more clear? Potentially something like `validFrom`?

Reply:
Yupz better. Suggest to change the name of issueDate as well is we are going to potentially use the issuanceDate from the Credential itself. We need something to define the startDate (valid From - valid To)

Reply:
LOL. We roughly responded at the same time. Yes validFrom validTo would be more clear IMO

Reply:
Went with `validFrom` and `validUntil`.